src/index.js

```diff
@@ -1,45 +1,54 @@
 import './style.css';
 
-const Tasks = [];
-let count = 0;
+// const updateLocalStorage = require('./storage.js');
 
+const renderTasks = require('./render.js');
+
+const storedTasks = JSON.parse(localStorage.getItem('tasks'));
+const tasks = storedTasks || [];
+
+// const list = document.getElementById('todoList');
+const addInput = document.getElementById('newTask');
+const addButton = document.getElementById('addBtn');
 class Todo {
-  constructor(description, completed, index) {
+  constructor(description, completed = false) {
     this.description = description;
     this.completed = completed;
-    this.index = index;
+    this.index = tasks.length + 1;
   }
-}
 
-Tasks.push(new Todo('Read', false, (count += 1)));
-Tasks.push(new Todo('Yoga', false, (count += 1)));
-Tasks.push(new Todo('Cook', false, (count += 1)));
+  addArray() {
+    tasks.push({
+      description: this.description,
+      completed: this.completed,
+      index: this.index,
+    });
+  }
+}
 
-const Items = document.createElement('div');
+if (storedTasks && storedTasks.length) {
+  renderTasks(storedTasks);
+}
 
-const addList = () => {
-  Tasks.forEach((task) => {
-    const taskItem = document.createElement('div');
+function createTask() {
+  const text = addInput.value;
 
-    const descriptionElem = document.createElement('span');
-    descriptionElem.innerText = task.description;
-    const checkBox = document.createElement('input');
-    checkBox.type = 'checkbox';
-    taskItem.classList.add('element');
-    taskItem.appendChild(checkBox);
-    taskItem.appendChild(descriptionElem);
-    Items.appendChild(taskItem);
-  });
+  if (text === '') {
+    return;
+  }
+  const task = new Todo(text);
 
-  document.getElementById('items').appendChild(Items);
+  tasks.push(task);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  renderTasks.renderTasks(tasks);
 
-  const removeButton = document.createElement('button');
-  removeButton.classList.add('removeBtn');
-  removeButton.innerText = 'Clear all completed';
-  removeButton.type = 'button';
-  Items.appendChild(removeButton);
+  addInput.value = '';
+}
 
-  return Items;
-};
+addButton.addEventListener('click', createTask);
 
-addList();
\ No newline at end of file
+addInput.addEventListener('keypress', (event) => {
```

Comment:
- [ ] This `eventListener` method on the `addInput` element is throwing a console error and the it is not functional. :pray: 
- [ ] Since it is not possible to check if the project requirements, please make sure all the project's requirements are met before you submit for another review. :pray: 
![Screenshot from 2021-11-05 21-12-23](https://user-images.githubusercontent.com/64894927/140559167-5bd1ca26-1355-4135-bfe7-abd7a49c4124.png)

Reply:
You did great. Now everything works fine! :pray: